src/jvm/main/org/jetbrains/kotlinx/lincheck/Utils.kt

```diff
@@ -277,4 +284,25 @@ private inline fun <R> runInIgnoredSection(currentThread: Thread, block: () -> R
         block()
     }
 
+/**
+ * Exits the ignored section and invokes the provided [block] in the ignored section, setting
+ * the [TestThread.inIgnoredSection] flag to `false` in the beginning and setting it back
+ * in the end to `true`.
+ * This method **must** be called in an ignored section.
+ */
+@Suppress("UnusedReceiverParameter")
+internal inline fun <R> ParallelThreadsRunner.runWithoutIgnoredSection(currentThread: TestThread, block: () -> R): R {
+    if (!currentThread.inTestingCode) {
```

Comment:
Can this function be called outside the testing code?

Reply:
Currently, we use it only in single place (for coroutine resumption). 
Here it is guaranteed that the thread is in testing code. 
So I changed the `if` to `require`.

Reply:
Oh, sorry, I was mistaken. This function can actually be called outside of testing code in case of stress strategy. Let's just leave this `if` as is.